src/pages/hyunjaeKim/Login/Login.js

```diff
@@ -1,8 +1,80 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Login.scss";
 
 class LoginHyunJae extends Component {
+  state = {
+    email: "",
+    password: "",
+  };
+
+  saveData = data => {
+    localStorage.setItem("TOKEN", data.TOKEN);
+  };
+
+  handleLogin = () => {
+    fetch("http://10.58.0.45:8000/users/login", {
+      method: "POST",
+      body: JSON.stringify({
+        ...this.state,
+      }),
```

Comment:
```suggestion
      body: JSON.stringify({ email, password }),
```
스프레드 연산자를 잘 활용해주셨으나 제 생각에는 state에 들어가는 내용이 꼭 body에 필요한 프로퍼티와 일치하지 않을 수 있기 때문에 state 전체를 넣어주는 건 부적절해보이네요. 어떤 속성들이 들어가는지 명시해주되 구조 분해 할당과 객체 단축 속성명 문법을 사용해주면 조금 더 간결하게 표현할 수 있을 것 같습니다.

Reply:
넵 필요한 부분만 구조분해할당하여 간결하게 정정하였습니다:)!